### PR TITLE
update comment section

### PR DESCRIPTION
**Pull request by Pchemist13**
I updated comments for clarity

### CODE REVIEW

File: test_paul_testing_training.py [hunk 1]
```diff
@@ -20,5 +20,8 @@ def test_show_html():
         html_string_from_file = readfile.read()
     assert html_string == html_string_from_file
 
+
+# git update
```
Comment (Tylersuard) [inline test_paul_testing_training.py]: Clean up this line

Use docstrings

Comment (Pchemist13) [reply test_paul_testing_training.py]: fixed

Comment (Tylersuard) [reply test_paul_testing_training.py]: LGTM!

**Review by Tylersuard — APPROVED** [review]
LGTM!

**Pull request merged**